css/index.css

```diff
@@ -0,0 +1,289 @@
+
+body {
+    font-family: 'Manrope', sans-serif;
+    font-weight: bold;
+    color: #555e6c;
+    font-size: 14px;
+}
+
+.container {
+    margin-left: 40px;
+    margin-right: 40px;
+    margin-bottom: 40px;
+}
+
+ul {
+    background-color: blue;
+    list-style-type: none;
+}
+
+li {
+    float: left;
+    padding: 8px 10px;
+}
+
+hr.new1 {
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #f4f3f6;
+    width: 100%;
+}
+
+.nav { 
+    float: left;
+    color: #ccd1d7;
+    font-weight: bold;
+    font-family: 'Manrope', sans-serif;
+}
+
+.patch { 
+    font-weight: bold;
+    color:#1a2939;
+}
+
+.depo {
+    color:#47a8d1;
+}
+
+.img {
```

Comment:
@indalgit again here, same mistake which you did for colors class naming... See every single class name should be self descriptive. Here **img** name isn't enough explainable uses of it's on element.